src/index.js

```diff
@@ -13,5 +13,9 @@ export default [
   'Writing tests takes x3 time than writing actual code.',
   'We need you to work on features. Not performance.',
   'Please do not worry a lot about quality. This is a prototype.',
+  'Since the tests dont pass, just skip them for this delivery build.',
+  'Just make a script to restart the server everyday at 3:00 AM, when nobody uses it, to get around memory leaks.',
```

Comment:
Minor changes: `Let's make a script to restart the server everyday at 3:00 AM (when nobody uses it) to get around memory leaks.`

Reply:
What about: 'Just make a script to restart the server everyday at 3:00 AM, to get around memory leaks.' ?

Reply:
Agreed, let's go with that.